release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
 link:https://access.redhat.com/errata/RHSA-2020:2820[RHSA-2020:2820] advisory.
+
+[[ocp-4-4-12]]
+=== RHBA-2020:2871 - {product-title} 4.4.12 Bug Fix Update
+
+Issued: 2020-07-13
+
+{product-title} release 4.4.12 is now available. The list of container images and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:2871[RHBA-2020:2871] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:2875[RHBA-2020:2875] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5217641[{product-title} 4.4.12 container image list]
+
+[[ocp-4-4-12-bug-fixes]]
+==== Bug Fixes
+
+* Not blocking having several listeners, on different protocols, on the same
+port for `ovn-octavia` driver. This was not supported previously on the
+`ovn-octavia` driver, but now it is supported and there is no need to block it.
+Several listeners on different protocols can be exposed in the same port. This
+means that it is possible to have, for instance, the DNS service to expose port
+53 in both TCP and UDP protocols when using `ovn-octavia`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1847558[*BZ#1847558*])
+
+* Previously, the CoreDNS forward plugin used a random server selection policy
+by default. As a result, clusters failed to resolve the OpenStack API hostname
+if given multiple external DNS resolvers. The plugin now uses DNS servers in the
+order they are provided.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1851267[*BZ#1851267*])
+
+* Previously, if Fluentd was deployed standalone using the CLO, it crashed due
+to missing configuration details. With this release, an empty Fluentd
+configuration is provided to enable Pods to start, and a status is added to
+inform users that manual intervention is required.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1851381[*BZ#1851381*])
+
+* During installation or upgrade, the `openshift-controller-manager` did not
+correctly report its progress condition. As a result, an installation or upgrade
+might fail. Now the Operator correctly reports its progress upon a successful
+installation or upgrade.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1852249[*BZ#1852249*])
+
+[[ocp-4-4-12-upgrading]]
+==== Upgrading
+
+To upgrade an existing OpenShift Container Platform 4.4 cluster to this latest release, see
+xref:../updating/updating-cluster-between-minor.html#update-upgrading-web_updating-cluster-between-minor[Updating a cluster by using the web console]
+for instructions.
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.3.3 or earlier, you
+must restart all Pods after the upgrade is complete.
+
+This is because the service CA is automatically rotated as of {product-title}
```

Comment:
s/This is because the service CA/The service CA

Reply:
Tabling this for now - want to take a single pass at any changes to the content of the boilerplate text later.